public/doc/openapi-3-v1.0.0.json

```diff
@@ -952,6 +1012,116 @@
                     }
                 }
             }
+        },
+        "/workflows/{id}/link": {
+            "post": {
+                "tags": [
+                  "Workflow"
+                ],
+                "summary": "Create a resource link to a given workflow",
+                "operationId": "linkWorkflow",
+                "description": "Create a resource link to a given workflow",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/id"
+                    }
+                ],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "description": "attributes to describe a resource for workflow linkage",
+                                "required": [
+                                    "obj"
+                                ],
+                                "properties": {
+                                    "obj": {
+                                        "$ref": "#/components/schemas/ResourceObject"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "description": "Parameters need to create a link",
+                    "required": true
+                },
+                "responses": {
+                    "200": {
+                        "description": "Success"
+                    },
+                    "400": {
+                        "$ref": "#/components/responses/400BadRequestResponse"
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/401UnauthorizedResponse"
+                    },
+                    "403": {
+                        "$ref": "#/components/responses/403ForbiddenResponse"
+                    },
+                    "404": {
+                        "$ref": "#/components/responses/404NotFoundResponse"
+                    },
+                    "422": {
+                        "$ref": "#/components/responses/422UnprocessableEntityResponse"
+                    }
+                }
+            }
+        },
+        "/workflows/{id}/unlink": {
```

Comment:
do not need `{id}`. I will create a separate service class to do unlinking.

Reply:
@bzwei When you unlink you would need to know which workflow you are unlinking? We would need ID for that

Reply:
no. unlink simply means an object no longer requires a workflow. We can detect which tag_name causes the linkage and delete the tag_name.